jans-core/script/src/main/java/io/jans/model/custom/script/type/authzen/DummyAccessEvaluationType.java

```diff
@@ -0,0 +1,37 @@
+package io.jans.model.custom.script.type.authzen;
+
+import io.jans.model.SimpleCustomProperty;
+import io.jans.model.authzen.AccessEvaluationRequest;
+import io.jans.model.authzen.AccessEvaluationResponse;
+import io.jans.model.custom.script.model.CustomScript;
+
+import java.util.Map;
+
+/**
+ * @author Yuriy Z
+ */
+public class DummyAccessEvaluationType implements AccessEvaluationType {
+
+    @Override
+    public boolean init(Map<String, SimpleCustomProperty> configurationAttributes) {
+        return true;
+    }
+    @Override
+    public boolean init(CustomScript customScript, Map<String, SimpleCustomProperty> configurationAttributes) {
+        return true;
+    }
+    @Override
+    public boolean destroy(Map<String, SimpleCustomProperty> configurationAttributes) {
+        return true;
+    }
+
+    @Override
+    public int getApiVersion() {
+        return 1;
+    }
+
+    @Override
+    public AccessEvaluationResponse evaluate(AccessEvaluationRequest request, Object context) {
```

Comment:
What if you want to return "diagnostics" or some other type of error detail about why the policy failed.

Reply:
@nynymike Script has to throw exception with all details inside. Example:
```
    @Override
    public AccessEvaluationResponse evaluate(AccessEvaluationRequest request, Object scriptContext) {

        if (request.getResource().getType().equalsIgnoreCase("file")) {
            throw new WebApplicationException(Response
                    .status(Response.Status.BAD_REQUEST)
                    .entity("{\n" +
                            "  \"error\": \"invalid_resource_type\",\n" +
                            "  \"error_description\": \"Resource type 'file' is not allowed.\"\n" +
                            "}")
                    .build());
        }
        ...
}        
```